jetty-core/jetty-client/src/main/java/module-info.java

```diff
@@ -16,6 +16,8 @@
     requires org.eclipse.jetty.alpn.client;
     requires org.slf4j;
 
+    requires org.eclipse.jetty.compression;
+    requires org.eclipse.jetty.compression.gzip;
```

Comment:
why is gzip a hard requirement?

Reply:
Because `jetty-client` supported `gzip` out of the box.
If we don't require it, then this support will be lacking by default, and would require applications to explicitly configure it.